<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>744</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>994</width>
    <height>744</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>994</width>
    <height>744</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>3dViewer_v1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource_files/res.qrc">
    <normaloff>:/ico/260835.ico</normaloff>:/ico/260835.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>271</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_info">
     <attribute name="title">
      <string>Info</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_fileName">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_fileName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>File name:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_chooseFile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Choosing file</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_location">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Location:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_numbVertices">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of vertices:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_numbEdges">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of edges:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_numbVertices">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>200</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_numbEdges">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>220</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_fileSize">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>File size:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_fileSize">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_setLocation">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mb</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_interaction">
     <attribute name="title">
      <string>Interaction</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_rebuild">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Rebuild!</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_scale">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <double>20.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_OX">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-20.000000000000000</double>
      </property>
      <property name="maximum">
       <double>20.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_OY">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-20.000000000000000</double>
      </property>
      <property name="maximum">
       <double>20.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_OZ">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-20.000000000000000</double>
      </property>
      <property name="maximum">
       <double>20.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_ROX">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-180.000000000000000</double>
      </property>
      <property name="maximum">
       <double>180.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_ROY">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>170</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-180.000000000000000</double>
      </property>
      <property name="maximum">
       <double>180.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_ROZ">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>200</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-180.000000000000000</double>
      </property>
      <property name="maximum">
       <double>180.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_scale">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Scale</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_OX">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Move OX</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_OY">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Move OY</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_OZ">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Move OZ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_ROX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rotation OX</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_ROY">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rotation OY</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_ROZ">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rotation OZ</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_settings">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_typeProjection">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>120</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Type of projection</string>
      </property>
      <widget class="QRadioButton" name="radioButton_parallel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>99</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Parallel</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_central">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>99</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Central</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_typeLine">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>120</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Type of line</string>
      </property>
      <widget class="QRadioButton" name="radioButton_solid">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Solid</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_dashed">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Dashed</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_edges">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Edges</string>
      </property>
      <widget class="QLabel" name="label_colorEdges">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Color of edges:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_thicknessEdges">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Thickness of edges:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_thicknessEdges">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <double>15.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_colorOfEdges">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>61</width>
         <height>32</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>change</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_vertices">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Vertices</string>
      </property>
      <widget class="QLabel" name="label_colorVertices">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Color of vertices:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_sizeEdges">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Size of vertices:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_sizeVertices">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <double>20.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_colorOfVertices">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>61</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>change</string>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBox_displayMethod">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>320</y>
        <width>103</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>circle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>square</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>none</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_displayMethod">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Display method:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_backgroundColor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Background color:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_colorOfBackGround">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>350</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>change</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_record">
     <attribute name="title">
      <string>Record</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_screenshot">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Screenshot</string>
      </property>
      <widget class="QRadioButton" name="radioButton_bmp">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>.bmp</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_jpeg">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>.jpeg</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_screenshot">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Screenshot</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_screencast">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>181</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Screencast</string>
      </property>
      <widget class="QTextBrowser" name="textBrowser_screencast">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>161</width>
         <height>71</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Gif animation settings:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;- resolution: 640x480;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;- FPS: 10;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;- duration: 5s.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_screencast">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Screencast</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="Opengl" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>691</width>
      <height>661</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Opengl</class>
   <extends>QOpenGLWidget</extends>
   <header>opengl.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource_files/res.qrc"/>
 </resources>
 <connections/>
</ui>
